### DISCUSSION TITLE
Welcome to form Discussions!

### DISCUSSION THREAD

**Discussion by vermaseren** [form-dev/form#533] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexmyczko** [comment]
When will a new version be tagged or released?

**Reply by jodavies** [reply]
Hi Alex, at the workshop this week we had a lot of discussions re: things to do before a release. I'll summarize my notes in #475 (probably next week) and from there a timeline can be formulated. Probably there can be both a 5.0 release and a 4.3.2 bugfix release.

**Reply by jodavies** [reply]
By the way, since the idea is that FORM5 will not support 32bit systems, what happens for the 32bit Debian package? That stays on "4.3.2" while 64bit Debian has 5.0 ?

**Reply by alexmyczko** [reply]
@jodavies i guess it will just fall out at some point, i only found these links for further information. do you know anyone still running i386 (32bit) software, or x32 (32bit binaries on 64-bit amd64)... check with https://popcon.debian.org/

https://www.reddit.com/r/linux/comments/18mrxjk/debian_end_of_i386_support_is_on_its_way/

https://release.debian.org/testing/arch_qualify.html

ps: if you gift me answered, i'll earn a github badge (2nd answer/discussion done :)

https://buildd.debian.org/status/package.php?p=form&suite=sid the upper part white, are the officially released architectures, the one below darker white inofficial architectures.
i guess i386 will fall out or stay, if it stays, form will just have no more binaries if it can't
be built anymore.
